projects/common/src/time/time-range.service.ts

```diff
@@ -61,8 +61,10 @@ export class TimeRangeService {
   }
 
   public refresh(): void {
-    const currentStringTimeRange = this.getCurrentTimeRange().toUrlString();
-    this.applyTimeRangeChange(this.timeRangeFromUrlString(currentStringTimeRange));
+    if (this.isInitialized()) {
+      const currentStringTimeRange = this.getCurrentTimeRange().toUrlString();
```

Comment:
nit: does it necessary this const here? does not better to pass the variable directly?

Reply:
it's pre-existing, I just wrapped it with the if check - but I would say it probably was written that way for readability to avoid nesting 3-4 function calls